src/libxtp/CMakeLists.txt

```diff
@@ -9,9 +9,10 @@ list(REMOVE_ITEM VOTCA_SOURCES ${NOT_VOTCA_SOURCES})
 
 get_filename_component(_PARENT_PATH "${CMAKE_CURRENT_BINARY_DIR}" DIRECTORY)
 
+message(STATUS: "PARENT_PATH: ${_PARENT_PATH}")
 add_library(votca_xtp  ${VOTCA_SOURCES})
 set_target_properties(votca_xtp PROPERTIES SOVERSION ${SOVERSION})
-set_target_properties(votca_xtp PROPERTIES VOTCA_SHARE ${CMAKE_SOURCE_DIR}/xtp/share)
+set_target_properties(votca_xtp PROPERTIES VOTCA_SHARE ${_PARENT_PATH}/../share)
```

Comment:
I would use `${PROJECT_SOURCE_DIR}/share` here.

Reply:
The problem with using `${PROJECT_SOURCE_DIR}/share` is doesn't contain the basisset. So,  I have created a temporal folder at [share](https://github.com/votca/xtp/blob/xtpshare/share/CMakeLists.txt) where I collect all the the data that need to be on `VOTCASHARE`.

Reply:
Where do they get created? `${PROJECT_BINARY_DIR}/share`?

Reply:
They XML files already exist in different locations of the `votca/xtp/share` folder that **do not correspond one to one with the final installation folder**. During the installation they are all put under share/xtp. So, I was wondering if I can use the installation `share` path because they are in the correct subfolders.

Reply:
I see, you need the xml files in`${VOTCASHARE}/xtp/xml`. How about moving `share/xml` to `share/xtp/xml`, then you can set `VOTCA_SHARE` above to `${PROJECT_SOURCE_DIR}/share`.

Reply:
moving the `share/basis_set` to `share/xtp/basis_set` made the trick, the jupyter notebooks now use the `${PROJECT_SOURCE_DIR}/share` folder as `VOTCASHARE` and run properly (at least locally)